android/uhabits-core/src/test/java/org/isoron/uhabits/core/io/HabitsCSVExporterTest.java

```diff
@@ -68,11 +68,11 @@ public void testExportCSV() throws IOException
         unzip(archive);
 
         assertPathExists("Habits.csv");
-        assertPathExists("001 Wake up early");
-        assertPathExists("001 Wake up early/Checkmarks.csv");
-        assertPathExists("001 Wake up early/Scores.csv");
-        assertPathExists("002 Meditate/Checkmarks.csv");
-        assertPathExists("002 Meditate/Scores.csv");
+        assertPathExists("001 Meditate/Checkmarks.csv");
+        assertPathExists("001 Meditate/Scores.csv");
+        assertPathExists("002 Wake up early");
+        assertPathExists("002 Wake up early/Checkmarks.csv");
+        assertPathExists("002 Wake up early/Scores.csv");
```

Comment:
Why has this changed? It seems unrelated to the feature.

Reply:
The default secondaryOrder is by name: https://github.com/iSoron/uhabits/pull/660/files#diff-36c07e8fa292800a1e6f010c52b7d81b33e2f1bc3514577d0c60df3d50959a86R42

Before my changes, this test was using the position sort. Given that these habits have the same position (`0`), they were left as they are inserted, so sorted by id. The changes reflect the facts that the habits are now sorted by name.